package.json

```diff
@@ -20,7 +24,7 @@
     "webrtc-adapter-test": "^0.2.7"
   },
   "scripts": {
-    "test": "eslint rtcstats.js",
+    "test": "./node_modules/.bin/eslint rtcstats.js",
```

Comment:
when running `npm test` node_modules/.bin/ should be in your path automagically?